asapdiscovery-docking/asapdiscovery/docking/scripts/run_docking_oe.py

```diff
@@ -101,30 +105,30 @@ def load_dus(file_base, by_compound=False):
         re_pat = r"([A-Z]{3}-[A-Z]{3}-[a-z0-9]+-[0-9]+)_[0-9][A-Z]"
     else:
         re_pat = r"(Mpro-[A-Za-z][0-9]+)_[0-9][A-Z]"
-    print(f"Loading {len(all_fns)} design units")
+    logger.info(f"Loading {len(all_fns)} design units")
     for fn in all_fns:
         m = re.search(re_pat, fn)
         if m is None:
             search_type = "compound_id" if by_compound else "Mpro dataset"
-            print(f"No {search_type} found for {fn}", flush=True)
+            logger.warning(f"No {search_type} found for {fn}")
             continue
 
         dataset = m.groups()[0]
         full_name = m.group()
         du = oechem.OEDesignUnit()
         if not oechem.OEReadDesignUnit(fn, du):
-            print(f"Failed to read DesignUnit {fn}", flush=True)
+            logger.warning(f"Failed to read DesignUnit {fn}")
```

Comment:
I would say this is an error

```suggestion
            logger.error(f"Failed to read DesignUnit {fn}")
```

Reply:
I agreed at first but I think for this script we'd rather have this as a warning because it means that the program won't quit if just 1 or 2 DUs don't load.